assets/css/components/_news-and-views.scss

```diff
@@ -0,0 +1,40 @@
+figure {
```

Comment:
Please use nested selectors. For example, css for `figure` and `figure img` at line 8 can be written as:

```
figure {
  // css for figure

  img {
    // css for figure.img
  }
}
```
This structure will give a clear picture of parent-children relationship.

The same comment applies to other selectors in this pull request.